.github/workflows/tests.yaml

```diff
@@ -87,9 +87,10 @@ jobs:
               timeout-minutes: 30
               run: |
                   ./scripts/run_in_build_env.sh \
-                  "./scripts/tests/run_test_suite.py run \
-                     --iterations 2 \
+                  "./scripts/tests/run_test_suite.py 
                      --chip-tool ./out/debug/standalone/chip-tool \
+                     run \
+                     --iterations 2 \
```

Comment:
Won't this fail? `--chip-tool` is an argument to `run`.

Reply:
It is inconsistent, however not sure if  we can fix it in any other way as the commands depend on the tool (listing available tests).

I could place all paths  top level (and they would not be used for things like list). How strongly do you feel  about changing this?

Reply:
Resolved: can followup if we think this is as important for user interaction.